packages/app/src/cli/services/dev/migrate-app-module.ts

```diff
@@ -88,15 +88,35 @@ export function getModulesToMigrate(
   }, [])
 }
 
-export async function migrateAppModules(
-  extensionsToMigrate: LocalRemoteSource[],
-  appId: string,
-  type: string,
-  remoteExtensions: RemoteSource[],
-  developerPlatformClient: DeveloperPlatformClient,
-) {
+export async function migrateAppModules(options: {
+  extensionsToMigrate: LocalRemoteSource[]
+  appId: string
+  type: string
+  remoteExtensions: RemoteSource[]
+  migrationClient: DeveloperPlatformClient
+  developerPlatformClient: DeveloperPlatformClient
```

Comment:
Why two clients? you only need the migrationClient here (which is always a PartnersClient)